TESTING.md

```diff
@@ -0,0 +1,99 @@
+# Test Strategy
+
+The overall test strategy can be described as *minimalistic*, rather than extensive. The goal of the tests is to verify that the application is deployed properly, so we will verify the main application features. 
+
+Decide the testing scope carefully, keeping in mind the maintainability of the tests and the value they provide. 
+
+### Test types
+
+* Performance testing
+* Resilience testing
+* Security testing
+* Smoke testing
+* Functional testing
+
+
+The tests described here are _deployment_ tests since their goal is to verify that the software is correctly deployed with all the inherent features. Both functional and non-functional characteristics are evaluated in these tests, focusing on the installation aspect. 
+
+* The core asset features work properly when deployed
+* There are no regression bugs 
+
+Before writing any test scenario, understand the primary purpose of the chart. Take a look at the  [the documentation about the chart under test](https://github.com/bitnami/charts/tree/master/bitnami) as well as the [docker image documentation](https://github.com/bitnami?q=docker&type=all&language=&sort=). This will give you as a solid base for creating valuable test scenario.
```

Comment:
```suggestion
The tests described here are _deployment_ tests since their goal is to verify that the software is correctly deployed with all the inherent features. Both functional and non-functional characteristics are evaluated in these tests, focusing on the installation aspect. 

* The core asset features work properly when deployed
* There are no regression bugs 

Before writing any test scenario, understand the primary purpose of the chart. Take a look at [the documentation about the chart under test](https://github.com/bitnami/charts/tree/master/bitnami) as well as the [docker image documentation](https://github.com/bitnami?q=docker&type=all&language=&sort=). This will give you a solid base for creating valuable test scenarios.
```